src/main/resources/application.properties

```diff
@@ -10,12 +10,14 @@ server.port=8080
 #server.ssl.keyStoreType: <PKCS12>
 #server.ssl.keyAlias: <tomcat>
 
-
-######## EI Backend
+######## EI Front-End
 ei.frontendServiceHost=localhost
 ei.frontendServicePort=8080
 ei.frontendContextPath=
 ei.useSecureHttpFrontend=false
+
+######## EI Default Back-End
+ei.backendServerName=default
```

Comment:
Is this variable really needed? Since whatever specified here is always the default EI Backend instance.

Reply:
I don´t see what variable you mean, but I guess it was the variable to tell if we have a default back end, yes it was already deleted.

Reply:
I mean this variable " ei.backendServerName=default". Since the serverAdress and port specified below here will always be the default EI Backend instance to use. So there is no need of that default variable/parameter in properties file.

Reply:
Then I figured out the correct variable, Github code markings is a bit confusing as it does not point to exact line. But it is already removed.

Reply:
Ok, Why its still here?

Reply:
Maybe the user would want another default name.
I thought you mean the variable that was before "useDefaultServer"

Reply:
Note that the name is displayed in the GUI so they may want to set their own name.